docs/apache-airflow-providers-airbyte/index.rst

```diff
@@ -128,5 +128,5 @@ Downloading official packages
 You can download officially released packages and verify their checksums and signatures from the
 `Official Apache Download site <https://downloads.apache.org/airflow/providers/>`_
 
-* `The apache-airflow-providers-airbyte 3.7.0 sdist package <https://downloads.apache.org/airflow/providers/apache-airflow-providers-airbyte-3.7.0.tar.gz>`_ (`asc <https://downloads.apache.org/airflow/providers/apache-airflow-providers-airbyte-3.7.0.tar.gz.asc>`__, `sha512 <https://downloads.apache.org/airflow/providers/apache-airflow-providers-airbyte-3.7.0.tar.gz.sha512>`__)
-* `The apache-airflow-providers-airbyte 3.7.0 wheel package <https://downloads.apache.org/airflow/providers/apache_airflow_providers_airbyte-3.7.0-py3-none-any.whl>`_ (`asc <https://downloads.apache.org/airflow/providers/apache_airflow_providers_airbyte-3.7.0-py3-none-any.whl.asc>`__, `sha512 <https://downloads.apache.org/airflow/providers/apache_airflow_providers_airbyte-3.7.0-py3-none-any.whl.sha512>`__)
+* `The apache-airflow-providers-airbyte 3.8.0 sdist package <https://downloads.apache.org/airflow/providers/apache-airflow-providers-airbyte-3.8.0.tar.gz>`_ (`asc <https://downloads.apache.org/airflow/providers/apache-airflow-providers-airbyte-3.8.0.tar.gz.asc>`__, `sha512 <https://downloads.apache.org/airflow/providers/apache-airflow-providers-airbyte-3.8.0.tar.gz.sha512>`__)
```

Comment:
Current template use old naming for sdist packages, it should be fixed by 
https://github.com/apache/airflow/pull/39341